ftw/showroom/js/src/showroom.js

```diff
@@ -173,6 +182,11 @@ module.exports = function Showroom(items = [], options) {
     checkArrows();
   }
 
+  function refresh() {
+    options.references = Array.prototype.slice.call(document.querySelectorAll(".showroom-reference"));
```

Comment:
In which case would you have an other class? This is just a convention. I think it doesn't matter which class we use.